src/app/store/reducers/Action.Constants.tsx

```diff
@@ -1,3 +1,5 @@
+export const BASE_API = "https://jsonplaceholder.typicode.com";
 export const USERS_FETCH_ALL = "users/fetchAll";
 export const TASKS_FETCH_ALL = "tasks/fetchAll";
```

Comment:
it's better to name such kinds of constants as USERS_FETCH_ALL_API, TASKS_FETCH_ALL_API, TASK_CREATE_API

the same for errors - UNKNOWN_ERROR_MESSAGE

the same for labels - EXAMPLE_TEXT_LABEL
the same for headings (h1, h2 etc) - EXAMPLE_NAME_HEADING